server/routes/visitors.ts

```diff
@@ -177,6 +177,13 @@ export default function routes(
 
       req.session.visitSessionData.visit = getSelectedSlot(req.session.slotsList, req.body['visit-date-and-time'])
 
+      const reservationId = await visitSessionsService.reserveVisit({
```

Comment:
Some cases I think we need to eventually handle here:
* reservation fails (e.g. slot now full). `try/catch` and then setting error message like "session full, try again..."? Theoretical at the moment as I think the back-end currently not checking capacity, etc!
* `reservationId` already set and user, e.g., goes back to this page from check details page:
  1. form re-posted with same slot - would duplicate booking be created? Could probably avoid by checking if chosen slot same as that already in session.
  2. form re-posted with new slot - looks like this would record the new `reservationId` and leave the previous reservation orphaned. Could either cancel existing reservation first or leave the orphaned one to 'time out' I the back-end? Probably one to discuss with Phil, Richard, etc.

Reply:
I've merged but will look at this next 👍